tests/v2/validation/pipeline/Jenkinsfile.multibranch.recurring

```diff
@@ -6,9 +6,9 @@ node {
     def jobName = "${JOB_NAME}"
     def config = env.CONFIG
     def filename = "config.yaml"
-    def hostnameSuffix = ["v2.7-head":"v27", "v2.8-head":"v28", "v2.9-head":"v29", "head":"main"]
-    def releaseBranchTagMap = ["v2.7-head":"release/v2.7", "v2.8-head":"release/v2.8", "v2.9-head":"release/v2.9", "head":"main"]
-    def tagChartMap = ["v2.7-head":"2.7.12", "v2.8-head":"2.8.8", "v2.9-head":"2.9.2", "head":"2.9.2"]
+    def hostnameSuffix = ["v2.7-head":"v27", "v2.8-head":"v28", "v2.9-head":"v29", "v2.10-head":"v210", "head":"main"]
+    def releaseBranchTagMap = ["v2.7-head":"release/v2.7", "v2.8-head":"release/v2.8", "v2.9-head":"release/v2.9", "v2.10-head":"release/v2.10", "head":"main"]
+    def tagChartMap = ["v2.7-head":"2.7.12", "v2.8-head":"2.8.11", "v2.9-head":"2.9.5", "v2.10-head":"2.10.1", "head":"2.10.1"]
```

Comment:
can't we remove 2.7 since its out of support now?

Reply:
I agree, addressed this one